README.md

```diff
@@ -340,7 +340,32 @@ We also handle type promotion by explicitly coding for it. This is done in the a
 ### Testing
 
 Running tests will generate a `test-report.html` in the root dir.
-**TODO** Explain test organizatian and expression tables
 
-Evaluating within a test should be done by calling `generalEvaluate`.
-This will test both sync and async implementation if possible. 
+The testing environment is set up to do a lot of tests with little code.
+The files responsible for fluent behaviour reside in `test/util`.  
+Most test can be run by running the `runTestTable` method in `test/util/utils.ts`.
+This method expects a TruthTable. Multiple test are run over a TruthTable (one for every line).
```

Comment:
TruthTable is not a class anymore (the file is still called that way). Anyway lets rename it to TestTable or smth?

Reply:
Seems like a decent name, sure! I wanted to change the file structure anyway so maybe I should just do that in this PR. (Changes we talked about in #34)